### PR TITLE
add runtime in the results container next to highscore

### LINKED ISSUE #40: Add runtime in Local Storage and show with High score
Add time next to high score
grafik

So it will be possible to achieve a new high score with the same amount of turns, but with a faster time.

### PR DESCRIPTION
**Pull request by barkha-gupta**
Fix #40 
* added a runtime next to highscore in the results container. 
* rendering celebration component for a new case: same highscore with less time

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ function App() {
   const [choiceTwo, setChoiceTwo] = useState(null);
   const [disabled, setDisabled] = useState(false);
   const [highScore, setHighScore] = useState(0);
+  const [runtime, setRuntime] = useState(0);
```
Comment (idna001) [inline src/App.js:24]: I think elapsedTime and setTime are setter that you can use.

**Review by idna001 — CHANGES_REQUESTED** [review]
Worked fine, found two little improvments

**Comment by idna001** [comment]
Also if want you can add your contribution in README file

File: src/App.js (hunk 1 above)
Comment (barkha-gupta) [reply src/App.js:24]: Ok, will work on this

**Comment by barkha-gupta** [comment]
Hii @idna001 , I have made the changes. 
* added the extra condition in the same if statement
* removed runtime setters 
Should i raise a PR?

**Review by idna001 — APPROVED** [review]

**Review by idna001 — APPROVED** [review]
Perfect, Thank you!

**Comment by barkha-gupta** [comment]
Thanks @idna001 for this opportunity.

**Pull request merged**